src/test/java/api/support/fixtures/ServicePointExamples.java

```diff
@@ -5,21 +5,36 @@
 class ServicePointExamples {
   static ServicePointBuilder basedUponCircDesk1() {
     return new ServicePointBuilder("Circ Desk 1", "cd1",
-        "Circulation Desk -- Hallway").withPickupLocation(Boolean.TRUE);
+        "Circulation Desk -- Hallway").withPickupLocation(Boolean.TRUE)
+        .withHoldShelfExpriyPeriod(30, "Days");
```

Comment:
The similar aspects for loan policies in tests are slightly cheeky and use the Period domain class.

This secondary situation where we have periods of time, might be an opportunity in the future, to introduce an explicit part of the test code that manages the construction of time periods.

What do you think? (again, not a change for today)

Reply:
Sounds like the right thing to do.